docs/local-setup/running-testnet.md

```diff
@@ -103,24 +84,36 @@ apt install -y git binutils-dev libcurl4-openssl-dev zlib1g-dev libdw-dev libibe
 Then run the script:
 
 ```bash
-git clone https://github.com/nearprotocol/nearcore.git
+git clone {-b <branch name from where you want to clone>} https://github.com/nearprotocol/nearcore.git
 cd nearcore
 ```
 
+Build the binaries using:
+```bash
+cargo build -p neard --{debug, release}
+```
+
+
 Finally:
 On MacOS or Linux
 
 ```bash
-./scripts/start_testnet.py --nodocker
+nearup {environment}net --nodocker --binary-path path/to/nearcore/target/{debug, release}
 ```
 
 On Ubuntu
 
 ```bash
-./scripts/start_testnet.py --nodocker
+nearup {environment}net --nodocker --binary-path path/to/nearcore/target/{debug, release}
```

Comment:
same as above

Reply:
`--nodocker` is not required here, i believe

Reply:
`nearup` documentation (https://github.com/near/nearup/edit/master/README.md) mentions usage of `--nodocker` the code in https://github.com/near/nearup/blob/master/main.py also appears to be using `--nodocker`. Leaving it here for now. Will change once confirmed.

Reply:
my mistake